docs/generators/spring.md

```diff
@@ -59,6 +59,7 @@ sidebar_label: spring
 |sortModelPropertiesByRequiredFlag|Sort model properties to place required parameters before optional parameters.| |true|
 |sortParamsByRequiredFlag|Sort method arguments to place required parameters before optional parameters.| |true|
 |sourceFolder|source folder for generated code| |src/main/java|
+|springBootVersion|Spring-boot version (will be used if library=spring-boot specified)| |2.3.0.RELEASE|
```

Comment:
Can this be done in another PR ? I don't think it adds much value and we already have a load of options...